packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -186,7 +186,7 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
       torrentInfoHash,
       peerAccount
     });
-    log('SEEDER: > unblockedPeer', peerAccount, 'from', Object.keys(this.peersList));
+    log('<< unblockedPeer', peerAccount, 'from', Object.keys(this.peersList));
```

Comment:
What's the thinking behind this notation?

Reply:
Everything which is preceded by `<<` it's done by the peer in a Leecher role.
Everything which is preceded by `>>` it's done by the peer in a Seeder role.
This allows me to filter what was important to me in multi-peer situations.